services/web/server/src/simcore_service_webserver/exporter/settings.py

```diff
@@ -24,6 +24,8 @@ class ExporterSettings(BaseCustomSettings):
 
 
 def get_settings(app: Application) -> ExporterSettings:
-    settings = app[APP_SETTINGS_KEY].WEBSERVER_EXPORTER
-    assert settings  # nosec
-    return settings
+    # settings = app[APP_SETTINGS_KEY].WEBSERVER_EXPORTER
+    # assert settings  # nosec
+    # return settings
+    # ANE -> PC: how was this working before?
+    return ExporterSettings()
```

Comment:
OK, I thought this was clear by but I see there are still doubts ... so let's talk next week and clarify still doubts.

In the meantime:
- ``ExporterSettings()``  works on its own because is a ``BaseCustomSettings`` so with the constructor it will capture the env settings
- Nonetheless, the new settings approach enforces all settings to be captured at the same time as fields of ``ApplicationSettings`` which is then saved in ``app[APP_SETTINGS_KEY].WEBSERVER_EXPORTER``
- Now,  if you did the test fixture right, both ``app[APP_SETTINGS_KEY].WEBSERVER_EXPORTER`` and ``ExporterSettings()`` should work